docs/getting_started.md

```diff
@@ -8,83 +8,103 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.8
 kernelspec:
-  display_name: 'Python 3.8.11 (''.venv'': venv)'
+  display_name: Python 3
   language: python
   name: python3
 ---
 
-+++ {"id": "9q9JZjt16AMf"}
++++ {"id": "6eea21b3"}
 
 [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/flax/blob/main/docs/getting_started.ipynb)
 [![Open On GitHub](https://img.shields.io/badge/Open-on%20GitHub-blue?logo=GitHub)](https://github.com/google/flax/blob/main/docs/getting_started.ipynb)
 
-# Getting Started
+# Quick Start
```

Comment:
Nit: Both "quickstart" and "quick start" seem to be OK. To be consistent with "JAX Quickstart", would it make sense to name our doc "Quickstart" (one word) or "Flax quickstart"?